internal/command/admin/admin.go

```diff
@@ -20,6 +20,10 @@ import (
 )
 
 type Admin struct {
-	Deploy deploy.Deploy `cmd:"" name:"deploy" aliases:"d" help:"deploy FunLess on different setups"`
-	User   user.User     `cmd:"" name:"user" aliases:"u" help:"create/delete FunLess users"`
+	Deploy deploy.Deploy `cmd:"" name:"deploy" aliases:"d" help:"Deploy FunLess on different setups"`
+	User   user.User     `cmd:"" name:"user" aliases:"u" help:"Create/delete FunLess users"`
+}
+
+func (f *Admin) Help() string {
+	return "Group of commands for managing and maintaining the platform"
```

Comment:
I think the standard help message for admin is enough and there is no need for the extra description, so you can remove the Help function here.